care/facility/api/serializers/patient.py

```diff
@@ -361,8 +360,12 @@ def update(self, instance, validated_data):
                 Disease.objects.bulk_create(diseases, ignore_conflicts=True)
 
             if meta_info:
-                for key, value in meta_info.items():
-                    setattr(patient.meta_info, key, value)
+                if patient.meta_info is None:
+                   meta_info_obj = PatientMetaInfo.objects.create(**meta_info)
+                   patient.meta_info = meta_info_obj
+                else:
+                    for key, value in meta_info.items():
+                        setattr(patient.meta_info, key, value)
```

Comment:
possibly confirm that we are not allowing to edit internal fields in this manner. ( eg, id, uuid, FK's and so on.. )